In [2]:
from scipy import stats

In [ ]:
stats.norm.cdf(70,60,10)

np.float64(0.8413447460685429)

In [ ]:
1 - stats.norm.cdf(70,60,10)

np.float64(0.15865525393145707)

In [ ]:
stats.norm.cdf(680,711,29)

np.float64(0.14254260383881612)

In [ ]:
1 - stats.norm.cdf(740,711,29)

np.float64(0.15865525393145707)

In [ ]:
stats.norm.cdf(750,711,29)

np.float64(0.9106595402534312)

In [ ]:
0.9106595402534312 - 0.14254260383881612

0.7681169364146151

# Normal distribution

The annual salaries of employees in a large company are approximately normally distributed with a mean of $50,000 and a
standard deviation of $10,000.

a) What percent of people earn less than $60,000?
![image-2.png](attachment:image-2.png)

In [4]:
from scipy.stats import norm
nd  = norm(50000,10000)
# nd stores a frozen normal distribution object
# It means that nd is an instance of a normal (Gaussian) distribution with:
# mean = 50000 and Std = 10000
# Internally, it is a rv_frozen object from SciPy,
# which means it is a ready-to-use probability distribution where the parameters are fixed.
# Now you can easily compute various functions like:
# nd.std(), nd.mean(), nd.var(), nd.cdf(), nd.ppf(), etc

x1 = nd.cdf(60000)
x1


np.float64(0.8413447460685429)

In [9]:
nd.mean() # try other functions

np.float64(50000.0)

In [3]:
# Same thing can be achieved using cdf()
stats.norm.cdf(60000,50000,10000)

np.float64(0.8413447460685429)

This means there's an 84.13% chance that a value from this distribution is less than or equal to 60,000.

# Example 2
A radar unit is used to measure speeds of cars on a highway. The speeds are normally distributed with a mean of 90 km/hr and a standard deviation of 10 km/hr.
What is the probability that a car picked at random is travelling at more than 100 km/hr?
![image.png](attachment:image.png)

In [ ]:
from scipy.stats import norm
nd  = norm(90,10)

x2 = 1 - nd.cdf(100)
x2

0.15865525393145707

# Example 3


For a certain type of mobiles, the length of time between charges of the battery is normally distributed with a mean of 10 hours and a standard deviation of 20 minutes. John owns one of these mobiles and wants to know the probability that the length of time will be between 10 and 11 hours.

![image.png](attachment:image.png)

In [ ]:
from scipy.stats import norm
nd  = norm(600,20)

x3 = nd.cdf(660) - nd.cdf(600)
x3

0.4986501019683699